test/uzal_cost_test.jl

```diff
@@ -1,91 +1,70 @@
 using DynamicalSystemsBase
 using DelayEmbeddings
 using StatsBase
-using DelimitedFiles
-using Revise
-using Random
-#using Plots
-using PyPlot
+using Test
+using Statistics
 
-pygui(true)
+println("\nTesting uzal_cost.jl...")
 
-## Simple Check on Lorenz System
-lo = Systems.lorenz()
-
-tr = trajectory(lo, 100; dt = 0.1, Ttr = 10)
+## Check on random vector
```

Comment:
to be on the safe side, its probably worth it to add a 
`Random.seed!(1516578735)` here.